.github/workflows

```diff
@@ -0,0 +1 @@
+.github/workflows/linters.yml
```

Comment:
- [ ] Linter should be set up correctly for this repository, the workflows should be a folder and inside it, you can add the linters.yml file

Reply:
Hi Ismaico. When I add forward slash while creating a file, it will push me to add another file. So should I leave the forward slash after workflows to make it as a folder?
Thanks
Amare